### PR TITLE
[CR] New Profession: Parkour Practitioner

### PR DESCRIPTION
**Pull request by railmonkey**
I already introduced this profession some months back on the forum. Now, for my first PR, I'd like to introduce you all to the Parkour Practitioner. It starts with the Parkour Expert trait and 4 Dodge, while cargo pants, a hoodie, and a runner pack give a nice bit of starting storage. Primarily a roleplay profession, but it's quite flexible and, naturally, synergizes quite well with the speed traits. Think of it as a straight upgrade to the Ballroom Dancer and an alternate option to the Martial Artist professions.

### CODE REVIEW

File: data/json/professions.json [hunk 1]
```diff
@@ -4445,5 +4445,42 @@
...
+	{
+        "type": "profession",
+        "ident": "parkour_practitioner",
+        "name": {
+            "male": "Traceur",
+            "female": "Traceuse"
+        },
+        "description": "You've practiced parkour for many years, and made the world your playground.  It wouldn't be a lie to say that running is your life.  Which is good, because now that the end has come, you're running for your life.",
+        "points": 2,
+		"traits": [
```
Comment (Coolthulhu) [inline data/json/professions.json]: Tabs

**Comment by chaosvolt** [comment]
Ah, interesting to see this being PR'd.

**Comment by railmonkey** [comment]
Heh. I figure I might as well. Less work to do it this way than to have to copy it in with every new experimental.

**Comment by Coolthulhu** [comment]
Does the cost include the cost for parkour expert?
The trait is really good and fits on every character, it is well worth the 2 points it costs.
4 dodge+hoodie+cargo pants are better than survivor starting gear (matches + pocket knife + warm but bulky armor + no skills).

**Comment by chaosvolt** [comment]
Hmm. Meanwhile the cost for 4 dodge in chargen is...well, far more than than the cost.

Though I'm not sure if the starting gear is straight-up better than the survivor's loadout. More storage (I think) and less encumbering, but it's a lot less warm and no tools. Then again, I tend to over-prioritize getting a comfy body temp when I play.

On the other hand, once you have a rock, you have all the starting tools you'll ever need. o3o

**Comment by railmonkey** [comment]
Yes, it does. I'm open to bumping it up, though, because yes, it is a really good trait. The reason why I went with 2 instead of anything further was precisely because it lacked that starting gear, and warmth is the biggest problem I ran into while running that build. So, would you say it's worth 3 points or 4? More than that?

**Comment by chaosvolt** [comment]
Granted, the starting gear is only a disadvantage under default considers, either spring start or scenario-induced winter start. In a summer start it becomes an advantage. XP

**Comment by Coolthulhu** [comment]
3 should probably be enough. You can get similar effects by taking ballroom dancer + one point in dodge + parkour expert. The only difference is starting equipment, but I don't think it's worth a whole extra point.

Hoodie + shirt is warm enough not to freeze in spring, assuming you've got shelter and aren't just roaming the wilds (and a parkour dude should be looting the city, not camping at LMOE). It is also good for combat, unlike survivor starting gear. Cargo pants are strictly better than the generic pants and together with hoodie give a good deal of storage.

**Comment by chaosvolt** [comment]
Hmm. True, though that does leave other parts of the body uncovered, but yeah. 1 point increases seems good.

**Comment by railmonkey** [comment]
This is true. Personally, I don't mess with the starting times, but that's just me. Initially, I had it at just 1 point for my personal use, since I was using it as a roleplay build without the fine consideration I had to give it once I put it in the public eye. Anyway, let me bump that number up really quick.

**Pull request merged**